components/radio/stories/radio.stories.js

```diff
@@ -91,31 +95,132 @@ export default {
 	},
 };
 
-export const Default = (args) => html`
+// ===== Story specific templates ======
+
+// @todo Radio needs more robust coverage of all states, including hover, focus, etc.
+const ChromaticGroupTemplate = (args) => html`
```

Comment:
I already left a similar comment on potentially moving all the markup to template.js. We could do that here too.